styx-service-common/src/test/java/com/spotify/styx/state/RunStateTest.java

```diff
@@ -310,243 +296,234 @@ public void testRetryDelayFromQueued() {
   }
 
   @Test
-  public void testRetryFromRunError() {
+  public void testRetryAfterFromRunError() {
```

Comment:
I think this test case meant to test a state after dequeue happens after `retryAfter`.